release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -317,6 +317,13 @@ With this release, the etcd tuning parameters can be set to values that optimize
 
 With this release, `dasd` image artifacts for the `s390x` architecture are removed from the {product-title} image building pipeline. You can still use the `metal4k` image artifact, which is identical and contains the same functionality.
 
+[id="egress-ip-etp-local-support"]
+=== Support for EgressIP with ExternalTrafficPolicy=Local services
+
+Previously, it was unsupported for EgressIP selected pods to also serve as backends for services with `externalTrafficPolicy` set to `Local`. When attempting this configuration, service ingress traffic reaching the pods was incorrectly rerouted to the egress node hosting the EgressIP. This affected how responses to incoming service traffic connections were handled and led to non-functional services when `externalTrafficPolicy` was set to `Local`, as connections were dropped and the service became unavailable.
+
+With {product-title} {product-version}, OVN-Kubernetes now supports the use of `ExternalTrafficPolicy=Local` services and EgressIP configurations at the same time on the same set of selected pods. OVN-Kubernetes now only reroutes the traffic originating from the EgressIP pods towards the egress node while routing the responses to ingress service traffic from the EgressIP pods via the same node where the pod lives.
```

Comment:
Suggest "where the pod is located" instead of "where the pod lives", do not want to introduce anthropomorphism.